people-counting-system/people-counting-client/src/app/device/device.page.html

```diff
@@ -7,12 +7,44 @@
   </ion-toolbar>
 </ion-header>
 
-<ion-content class="ion-padding" [fullscreen]="true">
-  <ion-header collapse="condense">
-    <ion-toolbar>
-      <ion-title size="large">Devices</ion-title>
-    </ion-toolbar>
-  </ion-header>
-  <!-- Your code goes here -->
-  <h1>Replace this with your markup</h1>
+<ion-content class="ion-padding">
+  <div style="display: flex; justify-content: flex-end; margin-bottom: 16px">
+    <ion-button color="primary" fill="outline">
+      <ion-icon
+        slot="start"
+        name="add-circle"
+        style="font-size: 24px"
+      ></ion-icon>
+      Add Device
+    </ion-button>
+  </div>
+
+  <div *ngFor="let device of devices$ | async" style="margin-bottom: 12px">
+    <ion-card style="margin: 0">
+      <ion-card-content style="padding: 8px 12px">
+        <div
+          style="
+            display: flex;
+            align-items: center;
+            justify-content: space-between;
+          "
+        >
+          <div>
+            <span> {{ device.name }} </span>
+          </div>
+          <div style="display: flex; gap: 8px">
+            <ion-button size="small" fill="outline" color="primary"
+              >View Counts</ion-button
+            >
+            <ion-button size="small" fill="outline" color="medium"
+              >Edit</ion-button
+            >
+            <ion-button size="small" fill="outline" color="danger"
+              >Delete</ion-button
+            >
+          </div>
+        </div>
+      </ion-card-content>
+    </ion-card>
+  </div>
```

Comment:
```suggestion
<ion-list class="ion-margin-top" [inset]="true" lines="inset">
    <ion-item *ngFor="let device of devices$ | async">
      <ion-label>{{ device.name }}</ion-label>
      <ion-button size="small" fill="outline" color="primary"
        >View Counts</ion-button
      >
      <ion-button size="small" fill="outline" color="warning">Edit</ion-button>
      <ion-button size="small" fill="outline" color="danger">Delete</ion-button>
    </ion-item>
  </ion-list>
```

Again get rid of the inline styles and use ion css classes.

Instead of the divs & cards use the ion list and ion item components